src/unittests/f32_interval.spec.ts

```diff
@@ -1129,3 +1140,61 @@ g.test('atan2Interval')
       `atan2Interval([${x}, ${y}]) returned ${got}. Expected ${expected}`
     );
   });
+
+g.test('multiplicationInterval')
+  .paramsSubcasesOnly<BinaryToIntervalCase>(
+    // prettier-ignore
+    [
+      // 32-bit normals
+      { input: [0, 0], expected: [0, 0] },
+      { input: [1, 0], expected: [0, 0] },
+      { input: [0, 1], expected: [0, 0] },
+      { input: [-1, 0], expected: [0, 0] },
+      { input: [0, -1], expected: [0, 0] },
+      { input: [1, 1], expected: [1, 1] },
+      { input: [1, -1], expected: [-1, -1] },
+      { input: [-1, 1], expected: [-1, -1] },
+      { input: [-1, -1], expected: [1, 1] },
+      { input: [2, 1], expected: [2, 2] },
+      { input: [1, -2], expected: [-2, -2] },
+      { input: [-2, 1], expected: [-2, -2] },
+      { input: [-2, -1], expected: [2, 2] },
+      { input: [2, 2], expected: [4, 4] },
+      { input: [2, -2], expected: [-4, -4] },
+      { input: [-2, 2], expected: [-4, -4] },
+      { input: [-2, -2], expected: [4, 4] },
+
+      // 64-bit normals
+      { input: [0.1, 0], expected: [0, 0] },
+      { input: [0, 0.1], expected: [0, 0] },
+      { input: [-0.1, 0], expected: [0, 0] },
+      { input: [0, -0.1], expected: [0, 0] },
+      { input: [0.1, 0.1], expected: [minusNULP(hexToF32(0x3c23d70a), 2), plusOneULP(hexToF32(0x3c23d70a))] },  // ~0.01
+      { input: [0.1, -0.1], expected: [minusOneULP(hexToF32(0xbc23d70a)), plusNULP(hexToF32(0xbc23d70a), 2)] },  // ~-0.01
+      { input: [-0.1, 0.1], expected: [minusOneULP(hexToF32(0xbc23d70a)), plusNULP(hexToF32(0xbc23d70a), 2)] },  // ~-0.01
+      { input: [-0.1, -0.1], expected: [minusNULP(hexToF32(0x3c23d70a), 2), plusOneULP(hexToF32(0x3c23d70a))] },  // ~0.01
+
+      // Infinities
+      { input: [0, Number.POSITIVE_INFINITY], expected: [0, 0] },
+      { input: [1, Number.POSITIVE_INFINITY], expected: [kValue.f32.positive.max, Number.POSITIVE_INFINITY] },
+      { input: [-1, Number.POSITIVE_INFINITY], expected: [Number.NEGATIVE_INFINITY, kValue.f32.negative.min] },
+      { input: [Number.POSITIVE_INFINITY, Number.POSITIVE_INFINITY], expected: [Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY] },
+      { input: [0, Number.NEGATIVE_INFINITY], expected: [0, 0] },
+      { input: [1, Number.NEGATIVE_INFINITY], expected: [Number.NEGATIVE_INFINITY, kValue.f32.negative.min] },
+      { input: [-1, Number.NEGATIVE_INFINITY], expected: [kValue.f32.positive.max, Number.POSITIVE_INFINITY] },
+      { input: [Number.NEGATIVE_INFINITY, Number.NEGATIVE_INFINITY], expected: [Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY] },
```

Comment:
Right. This line and the next should be   a single point at infinity, no?

Reply:
Done